executor/common.h

```diff
@@ -184,7 +204,11 @@ static void install_segv_handler()
 		*(type*)(addr) = new_val;                                         \
 	}
 
-#ifdef __NR_syz_emit_ethernet
+#if defined(__NR_syz_emit_ethernet) || defined(__NR_syz_extract_tcp_res)
+#define TUN_ENABLE
```

Comment:
Please prefix the macro name with SYZ_. We have all macros prefixed, and it's generally a good manner.